components/brave_adaptive_captcha/buildflags/buildflags.gni

```diff
@@ -1,3 +1,4 @@
 declare_args() {
-  brave_adaptive_captcha_enabled = is_mac || is_linux || is_win || is_ios
+  brave_adaptive_captcha_enabled =
+      is_mac || is_linux || is_win || is_ios || is_android
```

Comment:
Do we just say `true` in this case?

Reply:
oh yeah that makes sense. best way would be to remove the flag altogether as it supports all platforms.